release-notes/swan-lake-2201.0.4-release-note.md

```diff
@@ -15,6 +15,9 @@ redirect_from:
 
 ## Updating Ballerina
 
+>**info:** With this release, new updates of standard libraries and extensions are not available with previous releases. Update the current
```

Comment:
```suggestion
>**Info:** It is recommended to update your active Ballerina distribution to 2201.0.4 as soon as possible because from now onwards, new updates of standard libraries and extensions will not be available for the previous releases. 
```